admin/installer/setup_volumehub.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Set up Volume Hub if a token has been supplied.
```

Comment:
Probably, but all these scripts get appended to one long cloud-init script so we'd get mulitple copyright lines....which isn't the end of the world.
But also, we may end up replacing this "appended shell script" mechanism with a single Python module that contains all the necessary helper function and a single copyright header.
I'll leave it for now.